internal/kafka/command_topic_list.go

```diff
@@ -9,10 +9,18 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
-type topicOut struct {
+type topicNameOut struct {
 	Name string `human:"Name" serialized:"name"`
 }
 
+type topicOut struct {
+	Name              string `human:"Name" serialized:"name"`
+	Kind              string `human:"Kind" serialized:"kind"`
+	IsInternal        bool   `human:"Is Internal" serialized:"is_internal"`
```

Comment:
We use the same format for `IsCurrent`
```suggestion
	IsInternal        bool   `human:"Internal" serialized:"is_internal"`
```